x-pack/test/observability_functional/apps/observability/pages/alerts/custom_threshold.ts

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { Key } from 'selenium-webdriver';
+import expect from 'expect';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const testSubjects = getService('testSubjects');
+  const kibanaServer = getService('kibanaServer');
+  const supertest = getService('supertest');
+  const find = getService('find');
+
+  describe('Custom threshold rule', function () {
+    this.tags('includeFirefox');
+
+    const observability = getService('observability');
+    const dataView1 = 'filebeat-*';
+    const dataView2 = 'metricbeat-*';
+    const timeFieldName = '@timestamp';
+
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs');
+      // create two data views
```

Comment:
I created two data views because I wanted to deliberately opt into choosing one of them, rather than having it be selected by default, to confirm this behavior.